src/chai/ManagedArray.hpp

```diff
@@ -427,6 +427,8 @@ class ManagedArray : public CHAICopyable
  * \tparam T Type of the raw data.
  *
  * \return A new ManagedArray containing the raw data pointer.
+ *
+ * \note If using this method on HIP platforms, XNACK must be enabled.
```

Comment:
You could optionally add a line, "This is done by setting the HSA_XNACK environment variable to 1 or by compiling with :xnack+ appended to your AMD GPU target, see documentation at <link>"

Reply:
I added a link to their website.